cicd/docker/algod/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+set -e
+
+if [ $UPDATE_ON_START = true ]; then
+  if [ -z ${ALGOD_VERSION} ]; then
+    python3 /algod/setup.py install \
```

Comment:
no need to run the full `setup.py` script here, update.sh/find-nodes.sh should do the trick:
```
$HOME/node/update.sh $($HOME/node/find-nodes.sh /opt/testnetwork)
```

Also the ALGOD_VERSION bit doesn't look useful so this can be collapsed to a single command

Reply:
Cool, i'll make that change. Thanks!